markdownlint.js

```diff
@@ -182,9 +212,11 @@ program
   .version(pkg.version)
   .description(pkg.description)
   .usage('[options] <files|directories|globs>')
```

Comment:
Is .usage still needed after you add .arguments?

Reply:
Nope, that was me playing around with commander and forgot to take that out.  It looks like commander doesn't have a way to define that "everything else" is a specific parameter so basically none of the parameter validation (which I think for this just means required vs optional) actually does anything.  And regardless it looks like `.arguments()` is only for commands so it wasn't doing anything here.  I'll remove it.